app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ActiveRecord::Base
   def to_s
     case_id
   end
+
+  def wishlist
+    WishlistItem.includes(:course).where(user_id: id)
+  end
```

Comment:
Could we do this with a Rails association?
Maybe something like `has_many :wishlist, -> { includes(:course) }, class_name: 'WishlistItem'`